libraries/core_libs/consensus/include/vote_manager/vote_manager.hpp

```diff
@@ -280,7 +295,8 @@ class VoteManager {
   blk_hash_t reward_votes_block_hash_;
   PbftRound reward_votes_period_;
   PbftRound reward_votes_round_;
-  mutable std::shared_mutex reward_votes_info_mutex_;
+  // extra_reward_votes_ will only keep additional reward votes which are not part of the original 2t+1 cert votes
+  std::vector<std::shared_ptr<Vote>> extra_reward_votes_;
```

Comment:
based on this comment https://github.com/Taraxa-project/taraxa-node/pull/2352/files#r1136811150, I dont think you actually need this memory vector at all - you need to store those reward votes only in db and then load it during node init

Reply:
I removed it. The only downside of removing it is when we delete column in reward votes we will now delete with all the votes hashes. But from what I checked this is fast and it should not be a problem. Alternative would be to keep the vote hashes or extra reward votes only for deleting the db